packages/eslint-plugin/package.json

```diff
@@ -20,8 +20,7 @@
 		"url": "https://github.com/WordPress/gutenberg/issues"
 	},
 	"engines": {
-		"node": ">=14",
-		"npm": ">=6.14.4"
+		"node": ">=14"
```

Comment:
The same applies here as for `@wordpress/create-block`. Let's ensure the minimum npm version and there is no breaking change at this time.